readme.md

```diff
@@ -116,6 +116,9 @@ $ ava --help
 
 Files starting with `_` are ignored. This can be useful for having helpers in the same directory as your test files.
 
+*WARNING: NON-STANDARD BEHAVIOR:* The AVA CLI will always try to find and use your projects local install of AVA.
+ This is true even when you run the global `ava` command. This non-standard behavior solves an important [issue](https://github.com/sindresorhus/ava/issues/157),
+ and should have no impact on everyday use.
```

Comment:
Sure. Webstorm has only global and per file setting for soft wrapping. No ability to set for just `.md` etc. That's why I have gotten in that habit. 

https://github.com/nicoulaj/idea-markdown/issues/57